src/opera/test/pge/disp_s1/test_disp_s1_pge.py

```diff
@@ -678,7 +680,6 @@ def test_disp_s1_pge_validate_inputs(self):
                            ionosphere_files + troposphere_files)
         files_to_remove.append(mask_file)
         files_to_remove.append(dem_file)
-        files_to_remove.append(frame_to_burst_file)
```

Comment:
Since we're using the frame-geometries-simple file, and not the frame-to-burst database, should this line be put back?